cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/preview/PreviewHttpHandler.java

```diff
@@ -64,7 +69,18 @@
 @Path(Constants.Gateway.API_VERSION_3 + "/namespaces/{namespace-id}")
 public class PreviewHttpHandler extends AbstractLogHandler {
   private static final Gson GSON = new GsonBuilder()
-    .registerTypeAdapter(BasicThrowable.class, new BasicThrowableCodec()).create();
+    .registerTypeAdapter(BasicThrowable.class, new BasicThrowableCodec())
+    .registerTypeAdapter(ArtifactScope.class, new TypeAdapter<ArtifactScope>() {
```

Comment:
This factory will parse the enum value to lower case if we do GSON.toJson, which made us have to create a new GSON_STATUS for PreviewStatus, is that good?